code/game/machinery/cryopod.dm

```diff
@@ -269,10 +269,10 @@
 					reshape.Crop(7,4,26,31)
 					reshape.Crop(-5,-3,26,30)
 					GLOB.sac_image = reshape
-
-					for(var/datum/mind/H in SSticker.mode.cult)
-						if(H.current)
-							to_chat(H.current, "<span class='danger'>Nar'Sie</span> murmurs, <span class='cultlarge'>[occupant] is beyond your reach. Sacrifice [GLOB.sac_mind.current] instead...</span></span>")
+					if(GLOB.sac_mind == occupant)
+						for(var/datum/mind/H in SSticker.mode.cult)
+							if(H.current)
+								to_chat(H.current, "<span class='danger'>Nar'Sie</span> murmurs, <span class='cultlarge'>[occupant] is beyond your reach. Sacrifice [GLOB.sac_mind.current] instead...</span></span>")
```

Comment:
`GLOB.sac_mind` is already the new, picked target. You need to find a different way.